bitnami/concourse/templates/worker/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and .Values.worker.enabled .Values.worker.pdb.enabled -}}
```

Comment:
change pdb.enabled to pdb.create. It is mixed in our charts but pdb.create seems to have more ocurrences